scripts/tests/cirque_tests.sh

```diff
@@ -114,9 +114,7 @@ function cirquetest_bootstrap() {
     set -ex
 
     cd "$REPO_DIR"/third_party/cirque/repo
-    pip3 uninstall -y setuptools
-    pip3 install setuptools==65.7.0
-    pip3 install pycodestyle==2.5.0 wheel
+    pip3 install --break-system-packages pycodestyle==2.5.0 wheel
```

Comment:
Why do we not install a virtualenv instead? Why "break system packages" ?

Reply:
Because this is what cirque original script did. Updating this to be sane (virtualenv and likely to stop pulling 'latest' all the time) should be done as a maintenance pass rather than a hotfix.